### PR TITLE
make workqueue handle both reqmgr2 and reqmgr

### PR DESCRIPTION
**Pull request by ticoann**
@amaltaro, Alan, please review.

### CODE REVIEW

**Comment by amaltaro** [comment]
Looks good to me, though I have a couple of questions/comments. Will make it directly in the code.

File: src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py [hunk 1]
```diff
@@ -279,8 +282,13 @@ def _workQueueToReqMgrStatus(self, status):
                          'Done' : 'completed'
                          }
         if status in statusMapping:
+            # if wq status passed convert to reqmgr status
             return statusMapping[status]
+        elif status in RequestStatus.REQUEST_STATE_LIST:
```
Comment (amaltaro) [inline src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py]: Do we ever pass ReqMgr status to this method? If not, then we do not need to validate against ReqMgr2 statuses.

File: src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py [hunk 2]
```diff
@@ -251,11 +249,16 @@ def reportRequestStatus(self, request, status, message = None):
         """
         if message:
             self.sendMessage(request, str(message), 'info')
```
Comment (amaltaro) [inline src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py]: What is this message? When is it used? Is it the messages we see in ReqMgr when there is some failure? Depending on the answer, we could use logdb for this one as well :)

Comment (ticoann) [inline src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py]: sendMessage actually log to logDB it is wrapper function for reqmgr1 and reqmgr2 common calls.

File: src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py (hunk 1 above)
Comment (ticoann) [inline src/python/WMCore/WorkQueue/WorkQueueReqMgrInterface.py]: It does. line 180 above. Thanks for double checking.

**Pull request merged**